src/utils/pinned-entity-to-gltf.js

```diff
@@ -20,7 +20,14 @@ export default function pinnedEntityToGltf(el) {
   if (!equalArray(scale, [1, 1, 1])) gltfNode.scale = scale;
 
   if (components["media-loader"]) {
-    gltfComponents.media = { src: components["media-loader"].data.src, id: networkId };
+    const mediaSrc = components["media-loader"].data.src;
+
+    if (mediaSrc.startsWith("webrtc://")) {
```

Comment:
Ideally I shouldn't be able to pin them in the first place, but I am guessing that was a bit more of a pain to network.